common/src/main/java/io/novafoundation/nova/common/view/recyclerview/adapter/text/TextAdapter.kt

```diff
@@ -0,0 +1,54 @@
+package io.novafoundation.nova.common.view.recyclerview.adapter.text
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.annotation.StyleRes
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import io.novafoundation.nova.common.R
+import io.novafoundation.nova.common.utils.inflateChild
+import kotlinx.android.synthetic.main.item_text.view.itemText
+
+class TextAdapter(
+    initText: String? = null,
+    @StyleRes private val styleRes: Int = R.style.TextAppearance_NovaFoundation_Bold_Title2
+) : ListAdapter<TextRvItem, TextViewHolder>(TextAdapterDiffCallback()) {
```

Comment:
What is the reason to use ListAdapter here for 1 element? Can we use something similar like we do in `DAppHeaderAdapter`?